src/Tizen.NUI.Components/Attributes/DropDownAttributes.cs

```diff
@@ -306,12 +306,12 @@ public ImageAttributes CheckImageAttributes
         }
 
         /// <summary>
-        /// Right space from check image.
+        /// Gap of Check image to boundary.
```

Comment:
Is the boundary the border of the Control? If so then should it state which axis?  Could this not be done with Padding?